docs/examples/101/resource-with-lock-existing/main.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "accountName": {
+      "type": "string"
+    }
+  },
+  "functions": [],
+  "resources": [
+    {
+      "type": "Microsoft.Authorization/locks",
+      "apiVersion": "2016-09-01",
+      "scope": "[format('Microsoft.Storage/storageAccounts/{0}', parameters('accountName'))]",
```

Comment:
>scope [](start = 7, length = 5)

Resource locks support the scope property?

Reply:
Never mind - it's an extension resource.

---
In reply to: [568211082](https://github.com/Azure/bicep/pull/1339#discussion_r568211082) [](ancestors = 568211082)